### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by Fraasi**
I'm not that good in css, so there might be some improvements there, but this works perfectly and looks exactly like in the original site & using vanilla css variables to get the background colors etc.  

I somehow managed to break storybook on this one also, it seems that it doesn't get the background colors or fonts, propably because of using afore mentioned css variables, who knows?
Doesn't storybook have access to the 'root', or is it just for components completely in 'isolation'?  
If so, maybe it isn't good for this kind of components?  
You can see the footer in action in `npm start`, and all is good there.

### CODE REVIEW

File: src/components/Footer/index.css
Comment (meiamsome) [inline src/components/Footer/index.css:37]: I wonder if we should be using the verbose versions:
```css
.footer-links a.nature-of-code {
  background: #F35956;
  background: var(--ct-red, #F35956);
}
```

Sadly, it kind of ruins the nature of centralising the variables...

**Review by meiamsome — CHANGES_REQUESTED** [review]
The footer commit itself looks fine, I would suggest rebasing just 0aa81b onto master so it's merge-able, or waiting until #7 is merged.

**Comment by Fraasi** [comment]
I have not yet mastered the rebase spell, but I am trying to read about how to drop the unrelated commits from the header pull request which would fix this also. Stay tuned, but don't hold your breath in the meantime. (and now I'm sad, there's no wizard emoji [here](https://www.webfx.com/tools/emoji-cheat-sheet/). So here's a mailbox instead 📫 )

**Comment by Fraasi** [comment]
@meiamsome seems like my torture wasn't over just yet... 
I see we have a quite strict linting, wonder why my vscode-editor didn't warn about these 🤔   
Hopefully I got all the conflicts resolved ok.

Edit: Different linting rules in the project and in the circleci?

**Comment by Fraasi** [comment]
@meiamsome Okay, this rebasing thing is quite odd, but I'll get the hang of it. Everything should be now in order to merge this.

**Review by meiamsome — APPROVED** [review]

**Pull request merged**
